Lesson Plans/Flight Instructor/CFI/CFI1.2 Effective Communication.md

```diff
@@ -0,0 +1,109 @@
+		8. *As do all learners,* want to be shown respect
+		9. Need to increase or maintain self-esteem is a strong secondary motivator
+		10. Want to solve problems and apply new knowledge immediately
+	2. What instructors should do
+		1. Provide an organized training syllabus with clearly stated objectives
+		2. Help learners integrates new ideas *with what they already know*
+		3. Clarify and articulate all learner expectations early on
+		4. Recognize learner's need to control pace and start/stop time
+		5. Use frequent scenario-based training
+		6. Self-direction learning should still involve other people as resources, guides, etc.
+		7. Use books, programmed instruction and computers which are popular with adults
+		8. Refrain from "spoon-feeding"
+		9. Set a cooperative learning environment
+		10. Create opportunities for mutual planning
+2. Effective Communication
+	1. Definition
+		1. **Effective**: The similarity of the idea transmitted by the **source** and received by the **receiver**.
+		2. **Communication**: One person transmit ideas of feelings to another person or group.
+	2. Basic Elements:
+		1. **Source**: instructor, speaker, writer, etc.
+			1. The ability to select and use language is essential for transmitting meaningful symbols (ex. Don't use words like AD, ILS, TCAS to first-time learner)
+			2. Reveals information about themselves — self-image, their view of the idea being communicated and their feelings toward the receiver
+				1. Be confident; Illustrate the message is important and the receiver needs to know it.
+			3. Use accurate, up-to-date, and stimulating material
+				1. Old info lost you credibility; Boring info cause receiver's attention to be lost.
+		2. **Symbols**: Used to compose and transmit the message (words, signs and props, etc.)
+			1. Communication with symbols relies on perceptions, or channels: visual, auditory or kinesthetic. *Each learner may have different preference.*
+				1. ex. Instead of *telling* learners to trim the airplane, move the trim wheel for them and ask them to *feel* the difference on the control.
+			2. Instructor should constantly monitor feedback from the learner to identify misunderstandings and adapt presentation on-the-fly.
+			3. Instructor also should give learners feedback. Positive feedback builds learner's self-confidence and reinforces favorable behavior. *Negative feedback should be given in private and used carefully, be factual and nonjudgmental.*
+		3. **Receiver**: learner, reader, etc.
+			1. **Desired**: *Receiver reacts with understanding and changes their behavior according to the intent of the source.*
+			2. Learner's ability needs to be determined
+				1. Their motor skills level, aviation background / familiarity should be taken into consideration.
+			3. Background, cultural difference, education could all affect their behavior (willingness to accept, resistance, etc.)
+3. Barriers to Effective Communication
```

Comment:
Curse of Knowledge is another really interesting one. I'm going to add a page about it in a PR soon, but this Reel does a great job explaining it: https://www.instagram.com/reel/DOwD-JjgIQW/